client/providers/CallProvider/CallProvider.tsx

```diff
@@ -85,13 +84,15 @@ export const CallProvider: FC = ({ children }) => {
 	);
 
 	useEffect(() => {
-		Notifications.onUser('callerjoined', handleQueueJoined);
-		Notifications.onUser('agentcalled', handleAgentCalled);
-		Notifications.onUser('agentconnected', handleAgentConnected);
-		Notifications.onUser('queuememberadded', handleMemberAdded);
-		Notifications.onUser('queuememberremoved', handleMemberRemoved);
-		Notifications.onUser('callabandoned', handleCallAbandon);
-		Notifications.onUser('call.callerhangup', handleCallHangup);
+		if (voipEnabled) {
```

Comment:
What if i enable voip, the events get registered, and then i disable them? Will the event listeners be destroyed? 👀

Reply:
ok... fixing